provider/datagen/data/segmenter/lstm/lstm_km.json

```diff
@@ -1 +1 @@
288770676, -0.01896648108959198, -0.0241767056286335, -4.6535749435424805, -0.9893932342529297, 0.019423235207796097, 0.012999827973544598, 0.04400656372308731, -0.14550331234931946, 0.6895522475242615, 0.800469696521759, -0.39421018958091736, -1.0080537796020508, 0.06028655543923378, 0.1354190558195114, -1.2739872932434082, 0.3541595935821533, 0.5123111605644226, -0.516764760017395, -0.6702802777290344, -0.11612387001514435, -0.17625240981578827, -0.3047829568386078, -2.2642505168914795, 0.357505202293396, -0.18742482364177704, 0.22783170640468597, 1.3835617303848267, 0.08351948857307434, -0.09845644980669022, -0.9435850381851196, -0.7082817554473877, -0.842020571231842, 0.5601701736450195, -0.6748866438865662, 3.451209545135498, -2.9104626178741455, -0.848338782787323, 0.5990743041038513, 1.1752054691314697, -0.32512450218200684, -0.23160472512245178, -0.24709481000900269, -0.11124256253242493, -0.4071078300476074, -1.5432573556900024, 1.761533498764038, -2.3054962158203125, -1.6488847732543945, -1.2814785242080688, -0.16289688646793365, 2.018327474594116, 0.9978761672973633, -0.08259370923042297, 0.5467739701271057, -0.5080636143684387, -1.1563994884490967, 0.23326057195663452, -0.2975837290287018, -2.6421165466308594, 1.056286334991455, -0.28551235795021057, -0.21677082777023315, -0.6801796555519104, 0.5691878795623779, -0.07963666319847107, 0.37978339195251465, 1.875930666923523, -0.5361304879188538, 0.1980014443397522, -0.28615543246269226, -0.035504527390003204, 0.09134384989738464, 0.25882279872894287, -0.2167450338602066, -0.4472830295562744, 0.1421823501586914, 0.3439902663230896, 0.2037590593099594, 0.1389220803976059, -0.13854742050170898, -0.30649900436401367, 0.6859027743339539, 1.0516546964645386, 0.39112862944602966, 0.0014076000079512596, -0.9538235664367676, 0.8739601373672485, -0.5409058332443237, 1.4757333993911743, 0.6166747212409973, -0.3206327557563782, 1.0750422477722168, -1.170369029045105, -0.002706022933125496, -0.09527882933616638, -0.01115715317428112, 0.5480018854141235, 0.29330211877822876, -0.1806037425994873, 0.11832740902900696, -0.8801323771476746, 0.4812090992927551, -0.48577916622161865, 0.1450081169605255, -0.2678028643131256, -0.13597865402698517, -0.48129963874816895, -0.2577073872089386, 0.22190013527870178, 0.005426451563835144, 0.06588022410869598, 0.33103644847869873, -0.15126727521419525, 0.573759138584137, -0.13493692874908447, -0.6599060893058777, -0.14586462080478668, 0.1175009235739708, 0.016921747475862503, 0.24853789806365967, 0.2598995864391327, 0.05305597186088562, 1.5342333316802979, -0.28550484776496887, 0.3749963641166687, -1.0692145824432373, -0.2025602161884308, 0.9250209927558899, -0.18467006087303162, -0.19844625890254974, 0.14142858982086182, 0.1374323070049286, 0.5859553217887878, 1.09585440158844, -0.10420466214418411, 0.9840869307518005, -0.03364928066730499, -0.05966706573963165, 0.5782239437103271, -0.012656286358833313, -0.004029860720038414, -0.3935931921005249, -0.50538194179534
```

Comment:
I think this file is obsolete and unused. https://github.com/unicode-org/icu4x/pull/3235